### ISSUE TITLE
`npm run dev` does not work, after generating theme app extension

### ISSUE THREAD

**Issue by vpaul18** [Shopify/cli#1550]
### Please confirm that you have:

- [X] Searched [existing issues](.) to see if your issue is a duplicate. (If you’ve found a duplicate issue, feel free to add additional information in a comment on it.)
- [x] Reproduced the issue in [the latest CLI version](https://www.npmjs.com/package/@shopify/cli).

### In which of these areas are you experiencing a problem?

Extension, Theme

### Expected behavior

Run dev server through ngrok 

### Actual behavior

Hi!

I've seen other issue addressing this problem,but was stated that other issues should to be opened.

Here I get the following error :
Installing racc 1.6.2 with native extensions
Installing byebug 11.1.3 with native extensions

──────────────────────────────────────────────────────────────────────────────────────────────────────────

Error: Failed to kill process 7660: Error: Command failed: taskkill /pid 7660 /T /F
'taskkill' is not recognized as an internal or external command,
operable program or batch file.

Also, I tried to see how the dependencies are installed & no errors are being thrown with npm install.

What can I do to solve this issue ?
Thanks!

### Stack trace

```shell
at ./shop-theme-app/test-v1/node_modules/@shopify/cli-kit/dist/public/node/system.js:49:27
    at ChildProcess.exithandler (node:child_process:427:5)
    at ChildProcess.emit (node:events:513:28)
    at ChildProcess.emit (node:domain:489:12)
    at maybeClose (node:internal/child_process:1098:16)
    at Socket.<anonymous> (node:internal/child_process:456:11)
    at Socket.emit (node:events:513:28)
    at Socket.emit (node:domain:489:12)
    at Pipe.<anonymous> (node:net:317:12)
```

### Reproduction steps

1. npm init @shopify/app@latest
2. cd whatever-app
4. npm run shopify app generate extension
(installed ruby latest version)
5. npm run dev (npm run dev  --skip-dependencies-installation)

### Operating System

Windows 10

### Shopify CLI version (check your project's `package.json` if you're not sure)

"@shopify/app": "3.44.1", "@shopify/cli": "3.44.1"

### Shell

powershell

### Node version (run `node -v` if you're not sure)

v19.4.0

### What language and version are you using in your application?

Node

**Comment by Arkham** [comment]
hi @vpaul18, can you try again? we've released `3.45.0` which includes many fixes for Ruby on Windows.

**Comment by vpaul18** [comment]
@Arkham  Hi there! Sorry for the late reply, seems to have been fixed! Thanks!
